manifests/kustomize/third-party/argo/README.md

```diff
@@ -2,34 +2,33 @@
 
 Kubeflow Pipelines uses [Argo Workflows](https://argoproj.github.io/argo-workflows/) as the underlying workflow execution engine.
 
-This folder contains:
+This folder contains preconfigured Argo Workflows installations used in Kubeflow Pipelines distributions that use **remote references** to the upstream Argo Workflows repository instead of local copies.
 
-* `upstream/manifests` a mirror of argo workflows manifests upstream. It should never be edited here. Run `make update` to update it.
-* `installs` a folder with preconfigured argo workflows installations used in Kubeflow Pipelines distributions.
+Major differences from upstream argo manifests:
 
-  Major differences from upstream argo manifests:
+* Argo server is not included.
+* Argo workflow controller configmap is preconfigured to integrate with KFP.
+* Images are configured to use KFP redistributed ones which comply with open source licenses.
```

Comment:
Is this still true?

Reply:
Not true.
I removed this `Major differences` section entirely as it'll likely become outdated as we make more changes in the future.